.github/workflows/release.yml

```
@@ -20,9 +20,47 @@ concurrency:
+
+      - name: Determine whether electron-passkeys is about to publish
+        id: detect
+        run: node scripts/detect-electron-passkeys-publish.mjs >> "$GITHUB_OUTPUT"
+
+  build-native:
+    name: Build electron-passkeys native binaries
+    needs: detect-native
+    if: ${{ needs.detect-native.outputs.should-build == 'true' }}
+    permissions:
+      contents: read
+    uses: ./.github/workflows/electron-passkeys.yml
```

actionlint (1.7.12)

[error] error while parsing reusable workflow "./.github/workflows/electron-passkeys.yml": yaml: unmarshal errors:   line 19: mapping key "permissions" already defined at line 11

(workflow-call)